pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computeforwardingrule/regionalcomputeforwardingrule/_http.log

```diff
@@ -568,6 +477,7 @@ x-goog-request-params: project=${projectId}&region=us-central1
   },
   "loadBalancingScheme": "EXTERNAL",
   "name": "computeregionalforwardingrule-${uniqueId}",
+  "region": "projects/${projectId}/global/regions/us-central1",
```

Comment:
This field appears in TF-controller log, but not in direct controller log.

We have noticed this diff before, and I think this is a "bug" in TF-controller. `region` is [output only](https://screenshot.googleplex.com/EXdhuzf6oAYFNRU.png), it does nothing here in the request body. Also the format `global/regions` looks strange, I didn't find the source code of this value.